artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/cursor/impl/PageSubscriptionImpl.java

```diff
@@ -1444,20 +1276,39 @@ public List<MessageReference> getRelatedMessageReferences() {
 
    private class CursorIterator implements PageIterator {
 
-      private PagePositionAndFileOffset position = null;
+      // The cursorLogger is declared as static on the class, just to avoid a getLogger() call every time
+      private final Logger logger = cursorLogger;
```

Comment:
I would get this compilation error if I declared it static final:

Static declarations in inner classes are not supported at language level '11'


I wanted to use a different logger for the inner class...


but I will now just use the top logger, it's for tracing only anyways

Reply:
Would the earlier part of the suggestion, of just using the existing static variable you already declared at the top, still work?  (rather than reassigning it to the inner-class variable instance, which doesnt seem to achieve much)